.github/workflows/build_dev_man.yml

```diff
@@ -1,59 +1,45 @@
-name: Build and push for dev (Manually)
+name: Build and push for dev (Self-hosted)
```

Comment:
This name doesn't correlate with the filename "build_dev_man.yaml". May be build_dev_self_hosted.yaml?